lua/toggleterm/ui.lua

```diff
@@ -318,8 +318,7 @@ function M.open_split(size, term)
     if config.persist_size then M.save_window_size(term.direction, split_win.window) end
     api.nvim_set_current_win(split_win.window)
     local window_width = vim.o.columns
-    local horizontal_breakpoint =
-      require("toggleterm.config").get("responsiveness_settings").horizontal_breakpoint
+    local horizontal_breakpoint = config.get("responsiveness").horizontal_breakpoint
```

Comment:
The `.get(` is also unnecessary you can just do `config.responsiveness.horizontal_breakpoint`

Reply:
Done